assets/src/edit-story/components/helpCenter/useHelpCenter.js

```diff
@@ -86,12 +92,39 @@ const initial = {
     }),
     toggle: () => ({ isOpen }) => ({ isOpen: !isOpen }),
     close: () => () => ({ isOpen: false }),
+    fetchReadStart: () => () => ({ loading: true }),
+    fetchReadSuccess: (payload) => () => ({
+      ...payload,
+      loading: false,
+    }),
```

Comment:
Might help with legibility a bit to destructure the payload, so you can tell exactly what's coming in on this call. I'm guessing it's the `read` object?

Reply:
I think we should also alter this for the case where a user has slow network connection and is viewing tips before the hydrate/populate read tips call comes back:
```
    hydrateReadTipsSuccess: (payload) => (state) => ({
      readTips: {
         ...payload.readTips,
         ...state.readTips,
      },
      hydratingReadTips: false,
    }),
```